core/src/stored/bcopy.cc

```diff
@@ -216,7 +205,7 @@ int main(int argc, char* argv[])
 
   DeviceControlRecord* out_dcr = new DeviceControlRecord;
   out_jcr = SetupJcr("bcopy", output_archive.data(), bsr, director, out_dcr,
-                     outputVolumes, false); /* write device */
+                     outputVolumes.c_str(), false); /* write device */
```

Comment:
`SetupJcr` does a check to see if the volume name given as `const char*` is a valid pointer. Since the `.c_str()` always gives a valid pointer, the check doesn't make sense anymore since it will always pass.